src/Pages/3DPrinting/3D-print-form.css

```diff
@@ -1,13 +1,20 @@
-.App {
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@300&family=Mina:wght@400;700&family=Ubuntu:wght@700&display=swap');
```

Comment:
Is it possible we can use a different font that doesn't require a url? here are a bunch of css built-in fonts ([w3.org](https://www.w3.org/Style/Examples/007/fonts.en.html))